### ISSUE TITLE
Copying links doesn't work

### ISSUE THREAD

**Issue by MegaCarp** [lydell/LinkHints#65]
Alt-Shift-L Alt-<hint> doesn't copy link but does flash the copied part of the page. 

I am on linux however I reproduced the problem on a Windows machine as well.

I also reproduced the problem in a fresh firefox profile without addons except for LinkHint

It'd be helpful to find the logs, so that I may try and troubleshoot myself, however I couldn't find them after a cursory search - perhaps include that in the issues github template or even on the site?

### Debug info

<details>
<summary>Debug info</summary>

```json
{
  "version": "1.3.1",
  "browser": "firefox",
  "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0",
  "browserInfo": {
    "name": "Firefox",
    "vendor": "Mozilla",
    "version": "108.0.1",
    "buildID": "20221231063113"
  },
  "platformInfo": {
    "os": "linux",
    "arch": "x86-64"
  },
  "storage.sync": {},
  "storage.local": {
    "tutorialShown": true
  },
  "language": "en-US"
}
```

</details>

**Comment by lydell** [comment]
Hi!

The feature seems to work for me on macOS. Will need to check Linux and Windows as well later.

Regarding logs:

- In the Link Hints options, go to the bottom and expand “Debug”. There you can set the log level to Log or Debug to increase how much is logged. Note that it logs _a lot_ so it can be a bit difficult to filter through sometimes.
- Some logs appear in the “regular” page [JavaScript console](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#the_javascript_console).
- Some logs appear only in the “background”. Go to <about:debugging#/runtime/this-firefox>, expand Extensions and click Inspect on Link Hints.

**Comment by MegaCarp** [comment]
Hi! 

Good news: copying links as stated in the OP works now.

Bad news: I changed nothing, I just randomly decided to try it again and it worked. I went over the exact same test cases (google.com, test links within LinkHints options page) - they work now. This is on my main machine, my main firefox profile.

I find this annoying but I can't do anything about it, so for now I'm marking this closed and will get back to you if I ever stumble upon anything concrete.

Thank you for this absolutely stellar extension!
